@xen-orchestra/lite/src/components/pool/dashboard/AlarmRow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <tr v-if="isMobile" class="mobile" @click="handleCollapse">
+    <td v-if="isMobile" class="description" colspan="5">
+      <div v-tooltip class="ellipsis">
+        <ArrowCollapseButton :is-collapsed="!isDescriptionCollapsedRef" />
```

Comment:
I suppose it must have been
```suggestion
        <ArrowCollapseButton :is-collapsed="isDescriptionCollapsedRef" />
```

Anyway, according to Figma, it should be a `ButtonIcon`:
```suggestion
        <ButtonIcon :icon="isDescriptionCollapsed ? faAngleRight : faAngleDown" />
```